tools/Targets/Microsoft.Spot.system.gcc.targets

```diff
@@ -52,7 +52,7 @@
     <CC_CPP_TARGETTYPE_FLAGS Condition="'$(FLAVOR)'=='Instrumented'" >$(CC_CPP_TARGETTYPE_FLAGS) -O0</CC_CPP_TARGETTYPE_FLAGS>
     <CC_CPP_TARGETTYPE_FLAGS Condition="'$(FLAVOR)'=='Release'"      >$(CC_CPP_TARGETTYPE_FLAGS) -Os</CC_CPP_TARGETTYPE_FLAGS>
 
-    <CC_FLAGS>$(CC_FLAGS)-xc</CC_FLAGS>
+    <CC_FLAGS>$(CC_FLAGS)-std=c11 -xc</CC_FLAGS>

```

Comment:
My understanding is that switching to c11 adds only improvements in the compiling and linking. As the code base is "old" there are no new stuff brought with c11 that could break things. If that happens those issues can be corrected individually. Moving to c11 definitely improves the code size.